packages/neos-ui-editors/src/Editors/Button/index.js

```diff
@@ -0,0 +1,157 @@
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+import {Icon, IconButton} from '@neos-project/react-ui-components';
+import style from './style.module.css';
+
+import {neos} from '@neos-project/neos-ui-decorators';
+import {$transform} from 'plow-js';
```

Comment:
yes good catch. new feature code should not reintroduce plow and should be writting in native javascript. So `node.properties?.someProp` instead of `$get`.